tdrs-backend/tdpservice/parsers/schema_defs/ssp/m3.py

```diff
@@ -13,50 +13,144 @@
     preparsing_validators=[
         validators.notEmpty(start=19, end=60),
     ],
-    postparsing_validators=[],
+    postparsing_validators=[
+        validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.matches(1),
+                  result_field='SSN', result_function=validators.validateSSN(),
+            ),
+        validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                  result_field='RACE_HISPANIC', result_function=validators.isInLimits(1, 2),
+            ),
+        validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                  result_field='RACE_AMER_INDIAN', result_function=validators.isInLimits(1, 2),
+            ),
+        validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                  result_field='RACE_ASIAN', result_function=validators.isInLimits(1, 2),
+            ),
+        validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                  result_field='RACE_BLACK', result_function=validators.isInLimits(1, 2),
+            ),
+        validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                  result_field='RACE_HAWAIIAN', result_function=validators.isInLimits(1, 2),
+            ),
+        validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                  result_field='RACE_WHITE', result_function=validators.isInLimits(1, 2),
+            ),
+        validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                  result_field='RELATIONSHIP_HOH', result_function=validators.isInLimits(4, 9),
+            ),
+        validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                  result_field='PARENT_MINOR_CHILD', result_function=validators.oneOf((1, 2, 3)),
+            ),
+        validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.matches(1),
+                  result_field='EDUCATION_LEVEL', result_function=validators.notMatches(99),
```

Comment:
@jtimpe the language for this error is odd. please note this also applies to the first and second child in the record:

`if FAMILY_AFFILIATION :1 validator1 passed then EDUCATION_LEVEL 99 matches 99.`

this validator is checking that a recipient has a valid value for educ level. value 99 =="unknown"

Reply:
I agree the resulting error is confusing, but i think the implementation matches what you're saying - it errors because `EDUCATION_LEVEL` has a value of `99` and it wasn't supposed to.

@andrew-jameson @reitermb fyi - I can't find a ticket, but i remember at some point discussing wanting to research alternate error messages and update them later. i'm happy to discuss a change but i see us wanting to change a lot about the validator messages, especially cat3 and if/then validators. if there's a quick fix you think would be sufficient, let me know